src/app/pages/system/update/update.component.html

```diff
@@ -1,58 +1,153 @@
+        >
+          {{ 'Install Update' | translate }}
+        </button>
+      } @else if (showApplyPendingButton()) {
+        <button
+          *ixRequiresRoles="requiredRoles"
+          mat-button
+          color="primary"
+          ixTest="apply-pending-update"
+          (click)="applyPendingUpdate()"
         >
-          {{ 'Check Release Notes' | translate }}
-        </a>
+          {{ 'Apply Pending Update' | translate }}
+        </button>
+      }
+
+      @if ((updateService.status$ | async) === SystemUpdateStatus.Unavailable) {
+        <h4>
+          <b>{{ 'System is up to date!' | translate }}</b>
+          <ix-icon name="check_circle" class="green-color"></ix-icon>
+        </h4>
       }
-    </mat-card-content>
-  </mat-card>
-}
-
-@if ((updateService.changeLog$ | async) && (trainService.nightlyTrain$ | async)) {
-  <mat-card id="changelog-card">
-    <mat-card-title id="changelog-title">
-      <div>{{ 'Change log' | translate }}</div>
-    </mat-card-title>
-    <mat-card-content id="changelog-scrollbox">
-      <div [innerHTML]="updateService.changeLog$ | async"></div>
-    </mat-card-content>
-  </mat-card>
-}
-
-<ix-train-info-card></ix-train-info-card>
-<ix-update-actions-card></ix-update-actions-card>
+    </div>
+
+    @if ((updateService.status$ | async) === SystemUpdateStatus.RebootRequired) {
+      <h4 class="hint">{{ 'An update is already applied. Please restart the system.' | translate }}</h4>
+    }
+
+    @if (updateService.error$ | async) {
+      <h4 class="error-color">{{ updateService.error$ | async }}</h4>
+    }
+
+    @if (updateVersion()) {
+      <div class="update-summary">
+        <h3>
+          {{ 'Update Summary for {version}' | translate: { version: updateVersion() } }}
+        </h3>
+
+        @if (showInfoForTesting()) {
+          <p class="train-warning">
+            <ix-icon name="info"></ix-icon>
+            <strong>
+              {{ 'Selected train does not have production releases, and should only be used for testing.' | translate }}
+            </strong>
+          </p>
+        }
+
+        @if ((updateService.changeLog$ | async) && (trainService.nightlyTrain$ | async)) {
+          <h4>{{ 'Summary' | translate }}:</h4>
+
+          <div [innerHTML]="updateService.changeLog$ | async"></div>
+        }
+
+        @if (updateService.releaseNotesUrl$ | async; as releaseNotesUrl) {
+          <a
+            class="release-notes-link"
+            target="_blank"
+            ixTest="check-release-notes"
+            [href]="releaseNotesUrl"
+            [ixUiSearch]="searchableElements.elements.releaseNotes"
+          >
+            {{ 'Check Release Notes' | translate }}
+          </a>
+        }
+      </div>
+    }
+
+    <div class="other-options">
+      <h3>{{ 'Other Options' | translate }}</h3>
+
+      <h4>{{ 'Manual Update' | translate }}</h4>
+
+      <div class="manual-update">
+        <p>
+          <a
+            href="https://www.truenas.com/download/"
```

Comment:
Let's direct users to SCALE download page -> https://www.truenas.com/download-truenas-community-edition/